taskchampion/taskchampion/src/server/config.rs

```diff
@@ -37,7 +37,7 @@ pub enum ServerConfig {
         /// Bucket in which to store the task data. This bucket must not be used for any other
         /// purpose.
         bucket: String,
-
+        credential_path: Option<String>,
```

Comment:
Please add a docstring here. It's fine to link to a GCP docs page about credentials files.

Reply:
Ah I am extremely sorry, I should have added these docstrings in the previous review itself. Can you check [9eb49fd](https://github.com/akashShanmugraj/taskwarrior/commit/9eb49fd3153cf94c0d63224986e0bce86ecf369c) if the docstring is sufficient or too much?